File: node/prediction.js
const request = require('request');
const api = 'Enter-your-api-key'
const modelId = '';
const ImagePath = '';


const formData = {
    modelId,
    file: fs.createReadStream(ImagePath)
};
const resp = request.post({
    url: `https://app.nanonet.com/api/v2/ObjectDetection/Model/${modelId}/UploadFile/`,
    formData: formData,
    auth: {
        username: api,
        password: ''
    }
}, (err, body) => {
    callback(err, body)
});

File: python/prediction.py
import requests

model_id = 'Enter-your-model-id'
api_key = 'Enter-your-api-key'
image_path = 'path/to/your/image'

url = 'https://app.nanonets.com/api/v2/ObjectDetection/Model/' + model_id + '/LabelFile/'

data = {'file': open(image_path, 'rb'),    'modelId': ('', model_id)}

response = requests.post(url, auth=requests.auth.HTTPBasicAuth(api_key, ''), files=data)

print(response.text)